core/src/main/java/org/springframework/security/saml/SAMLProcessingFilter.java

```diff
@@ -107,6 +111,19 @@ public Authentication attemptAuthentication(HttpServletRequest request, HttpServ
 
     }
 
+	/**
+	 * Provided so that subclasses may configure what is put into the authentication
+	 * request's details property.
+	 *
+	 * @param request that an authentication request is being created for
+	 * @param authRequest the authentication request object that should have its details
+	 * set
+	 */
+	protected void setDetails(HttpServletRequest request,
```

Comment:
We don't want to introduce a new API in a maintenance release. Since no further minor releases are planned, let's leave this out.

Truthfully, it's likely not needed anyway since the user already gets full control by wiring an `AuthenticationDetailsSource`.